style.css

```diff
@@ -1,99 +1,727 @@
+}
+
+.card li {
+  display: inline-block;
+  padding: 5px 5px 5px 5px;
+  background-color: rgba(252, 252, 252, 0.247);
+  height: 40px;
+  color: rgba(255, 255, 255, 0.637);
+}
+
+.cardyellow {
+  background-color: #ff6b00;
+  height: 40px;
+  text-align: center;
+  padding-top: 10px;
+  color: #f1f1f1;
+  font-size: 20px;
+}
+
+#content_one {
+  background-color: #262626;
+  background-image: url(images/Frame.png);
+  background-repeat: no-repeat;
+  background-position: right bottom;
+  background-size: 30% 20%;
+  background-position-x: 100%;
+  background-position-y: 15%;
+  color: #fff;
+}
+
+#yellow button {
+  width: 160px;
+  height: 50px;
+  background-color: rgba(255, 107, 0, 1);
+  color: white;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 700;
   line-height: 24px;
+  letter-spacing: 0.03em;
+  padding-top: 5px;
+  border: none;
+  margin-left: 0;
+  margin-top: 20px;
+}
+
+.content_4 p {
+  font-size: 25px;
+  text-align: center;
+  padding: 20px;
+  color: #091e42;
+  font-weight: 100;
   font-family: "Times New Roman", Times, serif;
-  font-weight: normal;
-  color: #fff;
-  margin: 12px 0;
-  margin-left: 12px;
-  width: 100%;
+}
+
+.line5 {
+  background-color: #f8f8f8be;
+  height: 3px;
+  margin-top: 130px;
+  margin-bottom: 30px;
+  border-radius: 2px;
+}
+
+.bottom_link {
+  padding-bottom: 50px;
+}
+
+.bottom_link h2 {
+  margin-left: 35%;
+}
+
+.bottom_link ul {
+  align-content: center;
+  padding-left: 50px;
+  margin-bottom: 40px;
+  margin-top: 40px;
+  margin-left: 30%;
+  list-style-type: none;
+}
+
+.bottom_link li {
+  margin-top: 15px;
+  font-size: 20px;
+}
+
+.content_3 {
+  padding: 40px;
+}
+
+form {
+  padding: 20px;
+  margin: 10px;
+}
+
+form input {
+  margin-bottom: 10px;
+  width: 80%;
+}
+
+input[type="text"] {
+  padding: 12px 20px;
+  margin: 8px 0;
+  border: none;
+  border-bottom: 2px solid #dbd8d7;
+}
+
+input[type="email"] {
+  padding: 12px 20px;
+  margin: 8px 0;
+  box-sizing: border-box;
+  border: none;
+  width: 280px;
+  border-bottom: 2px solid #dbd8d7;
+}
+
+#fs-frm input[type="submit"] {
+  height: 48px;
+  background-color: rgba(255, 107, 0, 1);
+  color: white;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  padding-top: 13px;
+  border: none;
+  margin-left: 60px;
+  margin-top: 20px;
+  width: 150px;
+}
+
+textarea {
+  height: 150px;
+  width: 280px;
+  margin-top: 105px;
+  border: none;
+  background-color: #fbf8f7;
+  font-size: 16px;
+  padding-top: 1px;
+  padding-left: 2px;
+}
+
+footer {
+  margin: 40px;
+  height: 60px;
+  text-align: center;
+}
+
+.social-icons_last ul {
+  padding-top: 10px;
+}
+
+.social-icons_last li {
+  list-style: none;
+  display: inline;
+  height: 2ch;
+  font-size: 20px;
+  padding-right: 15px;
+  padding-top: 15px;
+}
+
+.social-icons_last a {
+  color: #505f79;
+}
+
+.form_view1 {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
```

Comment:
- [ ] The value for min-width property should be in range of 992px - 1100px. These are the accepted desktop screen breakpoints. Change your value. What you are using lies under the tablet screen breakpoints.